test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -1155,6 +1155,62 @@ public void TestReconnectionUndo()
             Assert.AreEqual(2, workspace.Connectors.Count()); 
         }
 
+        /// <summary>
+        /// The following recorded test manually adds a connection between 2 nodes,
+        /// followed by ctrl + clicking to copy this connection to 2 additional nodes.
+        /// The final assertion will only be true if all connections were successfully established.
+        /// </summary>
+        [Test, RequiresSTA]
+        public void TestCtrlClickInputConnections()
+        {
+            RunCommandsFromFile("CtrlClickRecordedTest_Recording.xml");
+
+            Assert.AreEqual(6, workspace.Nodes.Count());
+            Assert.AreEqual(7, workspace.Connectors.Count()); // 2 of which should be executed in the recording using ctrl + click
+
+            // Sum of values which is only true if connections are valid downstream
+            AssertPreviewValue("68d76b724b75499b8e96cfc106118107", 12);
+        }
+
+        /// <summary>
+        /// The following recorded test manually adds a connecting between 2 nodes,
+        /// followed by ctrl + clicking to copy this connection to 2 additional nodes.
+        /// We then call 2 undo commands and verify only our original connection exists.
```

Comment:
Nice, @alfarok would you point below where in the recorded tests we do `ctrl`? Just curious

Reply:
@QilongTang I think in the XML it just shows up as `MakeConnectionCommand` because `BeginDuplicateConnection` is a subset `MakeConnectionCommand`.  If you play back the recording or run the test you can see the way the connections are created is the same as in the .gif in the description though.

Reply:
@alfarok OK so we do not expose the level of details of key pressed but only commands in recorded tests, that is good to know